lib/awful/mouse/init.lua

```diff
@@ -298,7 +298,6 @@ end
 -- @treturn table The list of widgets.The first element is the biggest
 -- container while the last is the topmost widget. The table contains *x*, *y*,
 -- *width*, *height* and *widget*.
--- @see wibox.find_widgets
```

Comment:
For this comment, and all the others: this is what "make ldoc" spits before this commit:

/usr/src/awesome-build/lib/awful/widget/prompt.lua:69: no type specified, assuming function: 'awful.widget.prompt'
/usr/src/awesome-build/lib/awful/mouse/init.lua:303: current_widgets: function or section not found: wibox.find_widgets in wibox
/usr/src/awesome-build/lib/awful/mouse/init.lua:327: current_widget: function or section not found: wibox.find_widgets in wibox
/usr/src/awesome-build/objects/screen.c:162: screen.workarea: function or section not found: wibox.struts in wibox
/usr/src/awesome-build/lib/gears/timer.lua:190: gears.timer:start_new: module not found: timer
/usr/src/awesome-build/lib/gears/timer.lua:212: gears.timer:weak_start_new: module not found: timer


So, sure, wibox:find_widgets exists. But it doesn't have any docu, not even an empty one (like many signals). So I considered it an internal implementation detail, not an API.